elliottlib/cli/find_builds_cli.py

```diff
@@ -255,7 +260,7 @@ def _fetch_builds_from_diff(from_payload, to_payload, tag_pv_map):
     return _fetch_nvrps_by_nvr_or_id(nvrs, tag_pv_map)
 
 
-def _fetch_builds_by_kind_image(runtime, tag_pv_map, brew_session, p, np):
+def _fetch_builds_by_kind_image(runtime, tag_pv_map, brew_event, brew_session, p, np):
```

Comment:
we had it for RPMs already but not images? bizarre.

Reply:
Added it to rpm now